FordLabs/PeopleMover
# Make unit tests run faster in Jenkins

## Issue
Connects N/A

## What was done
- [x] Add jest flag to improve speed of unit tests in Jenkins (and theoretically help tests not be so flaky)

Resource:
https://jestjs.io/docs/en/troubleshooting#tests-are-extremely-slow-on-docker-andor-continuous-integration-ci-server

## How to test
1. Check out the time of the unit tests for this PR and confirm that they ran faster than normal.
